manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -1691,6 +1691,8 @@ spec:
           - cluster.local
           - --keepaliveMaxServerConnectionAge
           - "30m"
+          - --keepaliveInterval=30s
```

Comment:
Mesh config please ? Or env.

Reply:
It was already in the flag

Reply:
I think this pr is completing the charts